nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-properties-loader/src/test/resources/properties/conf/duplicates.nifi.properties

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-nifi.flow.configuration.file=./conf/flow.xml.gz
 nifi.flow.configuration.file=./conf/flow.json.gz
+nifi.flow.configuration.file=./conf/flow2.json.gz
```

Comment:
Is this change correct?

Reply:
Yes. According to the test uses this config, it is intentional to have two values for the same key